google/service_scope.go

```diff
@@ -26,8 +26,7 @@ func canonicalizeServiceScope(scope string) string {
 		"storage-ro":            "https://www.googleapis.com/auth/devstorage.read_only",
 		"storage-rw":            "https://www.googleapis.com/auth/devstorage.read_write",
 		"taskqueue":             "https://www.googleapis.com/auth/taskqueue",
-		"trace-append":          "https://www.googleapis.com/auth/trace.append",
-		"trace-ro":              "https://www.googleapis.com/auth/trace.readonly",
+		"trace":                 "https://www.googleapis.com/auth/trace.append",
```

Comment:
Thanks for upstreaming this here, sorry for the late feedback but I'm hesitant to make this change as we might break the users of trace-append and trace-ro. Do these fields not work at all as is? @rileykarson

Reply:
From my understanding this is just an alias. So could we not just add trace as an additional potion for trace.append instead of removing the other two

Reply:
Thank for reviewing, @ScottSuarez. Yes, as you say, it is just an alias but [our official document](https://cloud.google.com/sdk/gcloud/reference/alpha/compute/instances/set-scopes#--scopes) only mentions "trace" as the alias for `https://www.googleapis.com/auth/trace.append`, so terraform should support that alias as well.

I confirmed that "https://www.googleapis.com/auth/trace.readonly" is used still though it is not listed our [Google Compute Engine doc](https://cloud.google.com/sdk/gcloud/reference/alpha/compute/instances/set-scopes#--scopes) and [OAuth scope doc](https://developers.google.com/identity/protocols/oauth2/scopes).